Problem1.rb

```diff
@@ -0,0 +1,22 @@
+# sum of evem numbers from integer array
+
+def SumOfEven numbers =[]

```

Comment:
A very procedural way of writing this, but ruby is much more expressive and simple, can you rewrite this in one line?